src/tests/ov_helpers/ov_models/include/ov_models/utils/ov_helpers.hpp

```diff
@@ -10,8 +10,8 @@
 #endif
 
 #include <memory>
-#include <ngraph/opsets/opset1.hpp>
-#include <ngraph/runtime/tensor.hpp>
+#include <openvino/opsets/opset1.hpp>
+#include <openvino/runtime/tensor.hpp>
```

Comment:
- let's use `""` for OpenVINO headers
- can we avoid opsets and rely on concrete operations?